src/main/java/org/apache/accumulo/testing/performance/tests/ScanExecutorPT.java

```diff
@@ -75,10 +74,17 @@ public SystemConfiguration getSystemConfig() {
         SCAN_EXECUTOR_THREADS);
     siteCfg.put(Property.TSERV_SCAN_EXECUTORS_PREFIX.getKey() + "se1.prioritizer",
         SCAN_PRIORITIZER);
+    siteCfg.put(
+        Property.TSERV_SCAN_EXECUTORS_PREFIX.getKey() + "se1.prioritizer.opts.priority.se1p1", "1");
+    siteCfg.put(
+        Property.TSERV_SCAN_EXECUTORS_PREFIX.getKey() + "se1.prioritizer.opts.priority.se1p2", "2");
+
     siteCfg.put(Property.TSERV_SCAN_EXECUTORS_PREFIX.getKey() + "se2.threads",
         SCAN_EXECUTOR_THREADS);
     siteCfg.put(Property.TSERV_SCAN_EXECUTORS_PREFIX.getKey() + "se2.prioritizer",
         SCAN_PRIORITIZER);
+    siteCfg.put(
+        Property.TSERV_SCAN_EXECUTORS_PREFIX.getKey() + "se2.prioritizer.opts.priority.se2p1", "1");
```

Comment:
We can omit this if everything on the se2 pool will have the same prio, then it does not matter what it is. The default would be fine.

```suggestion
```